vite.config.ts

```diff
@@ -0,0 +1,18 @@
+import { vitePlugin as remix } from "@remix-run/dev";
+import { installGlobals } from "@remix-run/node";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+installGlobals();
```

Comment:
Is this necessary? I thought Node has all the happy things we like now.

Reply:
yeah good catch - if we're using Node 20+ then we can get rid of this and even if we keep it we would need to add the `installGlobals({ nativeFetch: true })` flag to work with single fetch.  I'll remove it when I get back to this 👍